components/async-status-mgt/org.wso2.carbon.identity.framework.async.status.mgt/src/main/java/org/wso2/carbon/identity/framework/async/status/mgt/api/buffer/SubOperationStatusObject.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023-2025, WSO2 LLC. (http://www.wso2.com).
```

Comment:
For the new classes, license year should be only the year it is introduced.
So this should be 2025
year range is applicable when modifying the older file

Reply:
Change all other newly added files' license header